README.md

```diff
@@ -13,26 +13,27 @@ const {
 } = utils;
 ```
 
-* **Remote Control** - The remote control utility requires iframe HTML Element with loaded Jitsi Meet.
+* **Remote Control** - The remote control utility requires iframe HTML Element that will load Jitsi Meet.
+
+Enable the remote control:
 ```Javascript
 const remoteControl = new RemoteControl(iframe/* the Jitsi Meet iframe */);
-...
-...
-...
-remoteControl.dispose();
 ```
-* **Desktop Sharing**
+
+To disable the remote control:
 ```Javascript
-setupScreenSharingForWindow(window/* the Jitsi Meet window object */);
+remoteControl.dispose();
 ```
-If Jitsi Meet is loaded in an iframe HTML element which is stored in the `iframe` variable, the code will look like:
+
+NOTE: `dispose` method will be called automatically when the Jitsi Meet iframe unload.
+
+* **Desktop Sharing**
 ```Javascript
-setupScreenSharingForWindow(iframe.contentWindow);
+setupScreenSharingForWindow(iframe/* the Jitsi Meet iframe */);
 ```
 
 ## Example
 
 For examples of installation and usage checkout the [Jitsi Meet Electron](https://github.com/jitsi/jitsi-meet-electron) project.
-
 ## Discuss
```

Comment:
Not related but noting here as a side thought: maybe this discussion section should be removed because it was also removed from the jitsi-meet readme.